versions.yaml

```diff
@@ -69,8 +69,12 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/qemu/qemu"
-      version: "stable-2.11"
-
+      version: "stable-2.12"
```

Comment:
Is this change required? If so, please put it into a separate commit explaining why it's changing.